fmt-tests/test-main/buildfile

```diff
@@ -0,0 +1,14 @@
+
+libs =
+import libs += fmt%lib{fmt}
+import libs += gtest%lib{gtest} gmock%lib{gmock}
+
+liba{test-main}: test/cxx{test-main gtest-extra util} test/hxx{gtest-extra mock-allocator test-assert util} $libs
```

Comment:
Would defining it as `libu` would have helped usage?

Reply:
Honestly, I'm yet to understand the intended usage of `libu*`, despite having read through the docs information on them many times. When/why do you choose to use it?

I also found this: ```Note also that if you plan to link any of your unit tests in the whole archive mode, then you will also need to exclude the source file containing the primary executable's main() from the utility library```
which suggests that if used here then `test-main` library could not contain `text-main.cc`. Having said that, the `test-main` library target is essentially reflecting upstream CMake and it's arguably not even worth having?